tests/scenario_test/test_get_hooks.py

```diff
@@ -1,12 +1,23 @@
 import json
 import runpy
+import sys
+from unittest.mock import patch
 
 from slack_cli_hooks.hooks import get_hooks, get_manifest, start
 
 
 class TestGetHooks:
+
+    def setup_method(self):
+        cli_args = [get_hooks.__name__, "--protocol", "message-boundaries", "--boundary", ""]
+        self.argv_mock = patch.object(sys, "argv", cli_args)
+        self.argv_mock.start()
+
+    def teardown_method(self):
+        self.argv_mock.stop()
+
     def test_get_manifest(self, capsys):
-        runpy.run_module(get_hooks.__name__, run_name="__main__")
+        runpy.run_module(get_hooks.__name__, run_name="__main__", alter_sys=False)
```

Comment:
Does this change need to be applied to other `runpy.run_module` methods? It seems like `False` is [the default value](https://docs.python.org/3/library/runpy.html) but it'd be nice to keep this consistent in whatever case IMO! 🙏

Reply:
Good catch 💯 this is a remnant of my experimental work